libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -106,35 +121,67 @@ export abstract class BaseInstallManager {
       throw new AlreadyReportedError();
     }
 
-    // Prevent update when using a filter, as modifications to the shrinkwrap shouldn't be saved
-    if (this.options.allowShrinkwrapUpdates && isFilteredInstall) {
+    // Ensure that subspaces is enabled
+
+    if (this.rushConfiguration.subspacesFeatureEnabled && !this.options.selectedSubspace) {
+      // Temporarily ensure that a subspace is provided
+      // eslint-disable-next-line no-console
+      console.log();
+      // eslint-disable-next-line no-console
+      console.log(
+        colors.red(
+          `The subspaces feature currently only supports installing for a specified set of subspace,` +
+            ` passed by the "--subspace" parameter or selected from targeted projects using any project selector.`
+        )
+      );
+      throw new AlreadyReportedError();
+    } else if (this.options.selectedSubspace && !this.rushConfiguration.subspacesFeatureEnabled) {
       // eslint-disable-next-line no-console
       console.log();
       // eslint-disable-next-line no-console
       console.log(
         colors.red(
-          'Project filtering arguments cannot be used when running "rush update". Run the command again ' +
-            'without specifying these arguments.'
+          `The "--subspace" parameter can only be passed if the "enabled" option is enabled in subspaces.json.`
```

Comment:
Bug: This error message gets printed when running `rush update-interactive` in a repository with the subspaces feature disabled.

Reply:
Also, is the `'Project filtering arguments cannot be used when running "rush update"`  validation being removed even in the case where the subspaces feature is disabled? Or was that check moved elsewhere.

As a test, try running `rush update` in this situation to see if the error message gets printed.

Reply:
I don't seem to get this error message when running `rush update-interactive`, how are you producing it?

Reply:
Running rush update with a filtered argument like `rush update -t .` yields the same error message as before:

```
usage: rush update [-h] [-p] [--bypass-policy] [--no-link]
                   [--network-concurrency COUNT] [--debug-package-manager]
                   [--max-install-attempts NUMBER] [--ignore-hooks]
                   [--offline] [--variant VARIANT] [--subspace SUBSPACE_NAME]
                   [--full] [--recheck]
                   
rush update: error: Unrecognized arguments: --to ..
```

Reply:
Repro steps:

1. Clone the https://github.com/microsoft/rush-example repository
2. `cd rush-example && testrush upgrade-interactive`
3. Choose "my-app"
4. Press SPACE to select "whatwg-fetch" for upgrading
5. Press ENTER to start the upgrade.

Reply:
We fixed this in e037b8f92563a200812f3997489e6c217952c75d